test/integration/apiserver/oidc/oidc_test.go

```diff
@@ -790,6 +796,8 @@ jwt:
 				Username: "k8s-john_doe",
 				Groups:   []string{"system:authenticated"},
 				Extra: map[string]authenticationv1.ExtraValue{
+					// validates credential id is set correctly and other extra fields are set
+					"authentication.kubernetes.io/credential-id": {"JTI=0123456789"},
 					"example.org/foo": {"bar"},
 					"example.org/baz": {"qux"},
 				},
```

Comment:
make sure we have a test case for a missing jti claim, and a non-string jti claim

Reply:
Unit tests cover all these scenarios and added an integration test to confirm non-string jti claim doesn't result in authentication errors.